.github/workflows/build.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: build
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
```

Comment:
Heh, by coincidence I just had to use this bit of python-fu elsewhere.  Small world.